packages/sui-react-web-vitals/package.json

```diff
@@ -17,7 +17,7 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "web-vitals": "3.5.2",
+    "web-vitals": "4.1.1",
```

Comment:
any breaking change we need to consider? 🤔

Reply:
![image](https://github.com/SUI-Components/sui/assets/18154356/fcbc113f-c8f7-4d5b-9708-458fd6417a06)

Reply:
There are breaking changes from version 2, but not for 3.